src/containers/HomeTemplate/Login/Login.js

```diff
@@ -0,0 +1,86 @@
+import React,{ useState} from 'react';
+import {useSelector,useDispatch} from "react-redux"
+import {FontAwesomeIcon} from "@fortawesome/react-fontawesome"
+import {faGooglePlus,faFacebook,faTwitter} from '@fortawesome/free-brands-svg-icons'
+import { faUser,faKey} from '@fortawesome/free-solid-svg-icons'
+import { actLogin } from './modules/actions';
+
+export default function Login(props) {
+    const error = useSelector((state)=> state.loginReducer.error)
+    const dispatch = useDispatch()
+  
+    const [state,setState] = useState({
+      taiKhoan:"",
+      matKhau:"",
+    })
+  
+    const handleOnchange = (event) => {
+      const {name,value} = event.target
+      setState({
+          ...state,
+          [name]:value ,
+      })
+  }
+  
+    const handleLogin = (event) => {
+      event.preventDefault();
+      dispatch(actLogin(state,props.history))
+    }
+  
+    const noti = () => {
+      return error && <div className='alert alert-danger'>{error.response.data}</div>
+    }
+  
+  
+      return <div className='loginUser'>
+                <div className="container">
+              <div className="d-flex justify-content-center h-100">
+                <div className="card">
+                  <div className="card-header">
+                    <h3>Sign In</h3>
+                    {noti()}
+                    <div className="d-flex justify-content-end social_icon">
+                      <span><FontAwesomeIcon icon={faGooglePlus}></FontAwesomeIcon></span>
+                      <span><FontAwesomeIcon icon={faFacebook}></FontAwesomeIcon></span>
+                      <span><FontAwesomeIcon icon={faTwitter}></FontAwesomeIcon></span>               
+                    </div>
+                  </div>
+                  <div className="card-body">
+                    <form onSubmit={handleLogin}>
+                      <div className="input-group form-group">
+                        <div className="input-group-prepend">
+                          <span className="input-group-text"><FontAwesomeIcon icon={faUser}/></span>
+                        </div>
+                        <input type="text" className="form-control" placeholder="Username" name="taiKhoan" onChange={handleOnchange} />
+                      </div>
+                      <div className="input-group form-group">
+                        <div className="input-group-prepend">
+                          <span className="input-group-text"><FontAwesomeIcon icon={faKey}/></span>
+                        </div>
+                        <input type="password" className="form-control" placeholder="Password" name="matKhau"  onChange={handleOnchange} />

```

Comment:
I would suggest you use useRef here, after user press login button. You will get the input values and fill them into the API calling function